src/javascript/ContentEditor.constants.js

```diff
@@ -74,5 +74,8 @@ export const Constants = {
         hexColorRegexp: /^#([a-f0-9]{3,4}|[a-f0-9]{4}(?:[a-f0-9]{2}){1,2})\b$/i,
         errorCode: 'invalidColor',
         selectorType: 'Color'
+    },
+    keyCodes: {
+        s: 83
```

Comment:
as it is for saving could we rename s to save?

Reply:
It's a key code for letter "s", maybe we should keep it this way to avoid confusion in the future? For escape I can put "esc" just like it appears on keyboard.